community/kernel/src/main/java/org/neo4j/kernel/impl/util/CoreRelationshipWrappingValue.java

```diff
@@ -25,21 +25,25 @@
 import org.neo4j.values.AnyValueWriter;
 import org.neo4j.values.storable.TextValue;
 import org.neo4j.values.storable.Values;
-import org.neo4j.values.virtual.RelationshipValue;
 import org.neo4j.values.virtual.MapValue;
 import org.neo4j.values.virtual.NodeValue;
+import org.neo4j.values.virtual.RelationshipValue;
 import org.neo4j.values.virtual.VirtualNodeValue;
 import org.neo4j.values.virtual.VirtualValues;
 
-public class RelationshipProxyWrappingValue extends RelationshipValue
+/**
+ * This RelationshipValue wraps a Core API Relationship and delegates method calls to the
+ * Relationship instance.
+ */
+public class CoreRelationshipWrappingValue extends RelationshipValue
```

Comment:
The `RelationshipProxy` objects were never the problem, and the problematic `Relationship` implementations we still wrap - so we gain nothing by this change...?